scrapy/http/request/form.py

```diff
@@ -172,6 +172,8 @@ def _get_clickable(clickdata, form):
         el for el in form.xpath(
             'descendant::*[(self::input or self::button)'
             ' and re:test(@type, "^submit$", "i")]'
+            '|descendant::*[(self::input or self::button)'
+            ' and re:test(@type, "^image$", "i")]'
```

Comment:
Considering `button[type=image]` isn't valid, I would go for:

```
descendant::input[re.test(@type, "^(submit|image)$", "i")]
|descendant::button[not(@type) or re.test(@type, "^submit$", "i")]
```

Reply:
Done! @dangra, please see the new changes.